### PR TITLE
Adding Kubernetes Native Configuration Management (kustomize.io) scripts

### PR DESCRIPTION
**Pull request by computate**
Kubernetes Native Configuration Management (kustomize.io) that builds upon your existing Helm Charts to install the orion-ld context broker on OpenShift Local. 

I have added documentation to setup OpenShift Local, and run the oc commands to install everything. The secrets are handled a little differently than your documentation. In the new kustomize configuration, it sets up a Hashicorp Vault in OpenShift which allows you to setup your MongoDB passwords and Red Hat SSO client secrets in a user interface, and the External Secrets Operator references the vault and sets up actual Secrets automatically. It also sets up ArgoCD and deploys mongodb and orion-ld as ArgoCD Applications.

### CODE REVIEW

File: kustomize/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- In the "fiware" path, add a new secret with a path of "mongodb-secret"
+- Add a password for the key: mongodb-password
+- Add a password for the key: mongodb-replica-set-key
+- Add a password for the key: mongodb-root-password
+- Click [ Save ]
+
+## Setup new argocd-auth-secret in vault
+
+- In Vault, click Secrets
+- In the "fiware" path, add a new secret with a path of "argocd-auth-secret"
```
Comment (wistefan) [inline kustomize/README.md:102]: I'm still working on it, but why  do you connect the local argo instance to you're (private?) keycloak instance? Wouldn't it be easier to simply delegate the auth for argo to openshift?

Comment (computate) [reply kustomize/README.md:102]: I am open to suggestions here for sure. I was hoping to build a consistent authentication for all the components (already deployed Red Hat SSO) for ArgoCD, OpenShift, Vault and the FIWARE applications. I figured logging into ArgoCD with the kubeadmin wasn't ideal or something like that. But maybe I'm just not familiar yet on how to make OpenShift Auth the best choice. Feel free to change. With Kustomize you can always apply patches per overlay and extend to do it your own way custom. Maybe OpenShift by default can use the Fiware provided Keycloak instance for authentication everywhere. What do you think?

Comment (wistefan) [reply kustomize/README.md:102]: @computate Hi, that sounds like a good idea. I take a look at that. Did you see my PR to your repo? I'm not able to add you as reviewer.

Comment (computate) [reply kustomize/README.md:119]: @wistefan Thanks for your commits to the PR from the FIWARE perspective. I have merged them into this branch!

**Review by wistefan — APPROVED** [review]

**Pull request merged**